docs/en_US/Tutorial/QuickStart.rst

```diff
@@ -128,55 +118,83 @@ Three steps to start an experiment
 
 *Example:* :githublink:`mnist.py <examples/trials/mnist-pytorch/mnist.py>`
 
-**Step 3**\ : Define a ``config`` file in YAML which declares the ``path`` to the search space and trial files. It also gives other information such as the tuning algorithm, max trial number, and max duration arguments.
+
+Step 2: Define the Search Space
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Define a ``Search Space`` in a YAML file, including the ``name`` and the ``distribution`` (discrete-valued or continuous-valued) of all the hyperparameters you want to search.
 
 .. code-block:: yaml
 
-   authorName: default
-   experimentName: example_mnist
-   trialConcurrency: 1
-   maxExecDuration: 1h
-   maxTrialNum: 10
-   trainingServicePlatform: local
-   # The path to Search Space
-   searchSpacePath: search_space.json
-   useAnnotation: false
-   tuner:
-     builtinTunerName: TPE
-   # The path and the running command of trial
-   trial:
-     command: python3 mnist.py
-     codeDir: .
-     gpuNum: 0
+   searchSpace:
+      batch_size:
+         _type: choice
+         _value: [16, 32, 64, 128]
+      hidden_size:
+         _type: choice
+         _value: [128, 256, 512, 1024]
+      lr:
+         _type: choice
+         _value: [0.0001, 0.001, 0.01, 0.1]
+      momentum:
+         _type: uniform
+         _value: [0, 1]
```

Comment:
it is a little strange that you are supposed to demonstrate "1. Write a config file and start the experiment from the command line.", but the format of this search space is for "launch from python"...

Reply:
and if you are demonstrate the first approach, suggest to use "config.yml" rather than "config_detailed.yml"

Reply:
This is inline(aka. embedded) search space, which can be defined in the YAML file, instead of the JSON file. [They](https://github.com/microsoft/nni/pull/3861#discussion_r658465990) think this approach is more convenient, for example, users only need to write this additional file, and can comment out each line at will. Only "config_detailed.yml" adopts the inline search space, in consistency, I suggest using this file.

Reply:
got it, thanks